cs/src/core/Bond.csproj

```diff
@@ -1,101 +1,36 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath32)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath32)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
-  <PropertyGroup Condition="'$(BuildNonportable)' == 'true'">
-    <BuildFramework>net45-nonportable</BuildFramework>
-    <DefineConstants>$(DefineConstants);SUPPORTS_XMLRESOLVER</DefineConstants>
-  </PropertyGroup>
-  <Import Condition="'$(BuildNonportable)' == 'true'" Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Common.Internal.props" />
-  <Import Condition="'$(BuildNonportable)' != 'true'" Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Portable.Internal.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Common.Internal.props" />
+
   <PropertyGroup>
-    <ProjectGuid>{43CBBA9B-C4BC-4E64-8733-7B72562D2E91}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Bond</RootNamespace>
-    <AssemblyName>Bond</AssemblyName>
+    <TargetFrameworks>net45;netstandard1.0;netstandard1.6</TargetFrameworks>
+    <Configurations>Debug;Release</Configurations>
     <BondRedistributable>true</BondRedistributable>
-    <DependentOutputPath>bin\$(BuildType)\net45</DependentOutputPath>
     <HasNonportableVersion>true</HasNonportableVersion>
```

Comment:
Are we using `HasNonportableVersion` any longer? I cannot find any consumers of this property.
Same comment for the other places it appears.